src/test/java/com/galvanize/alpha/speedwaytrials/racecars/RaceCarIT.java

```diff
@@ -23,11 +26,30 @@
     @Autowired
     MockMvc mockMvc;
 
+    @Autowired
+    ObjectMapper objectMapper;
+
     @Test
     public void getAllCars_EmptyList() throws Exception {
         mockMvc.perform(get("/cars"))
                 .andExpect(status().isOk())
                 .andExpect(jsonPath("length()").value(0))
                 .andDo(document("GetCars"));
     }
+
+    @Test
+    public void addCars() throws Exception {
+
+        CarsDto carsDto = new CarsDto("Ferrari");
+
+        mockMvc.perform(post("/cars").content(objectMapper.writeValueAsString(carsDto)).contentType(MediaType.APPLICATION_JSON))
+            .andExpect(status().isCreated())
+            .andDo(document("AddCars"));
+
+        mockMvc.perform(get("/cars"))
+            .andExpect(status().isOk())
+            .andExpect(jsonPath("length()").value(1))
+            .andExpect(jsonPath("[0].model").value("Ferrari"))
+            .andDo(document("GetCars"));
```

Comment:
Did you want to document the fields that you are getting back? I'm surprised this didn't throw and error, but maybe that is due to `responseFields` not being added so it doesn't ensure fields are documented.